src/Sulu/Bundle/ContactBundle/Controller/AccountController.php

```diff
@@ -82,12 +73,13 @@ class AccountController extends AbstractContactController
     protected $accountContactFieldDescriptors;
     protected $accountAddressesFieldDescriptors;
 
-    public function __construct()
+    public function setContainer(ContainerInterface $container = null)
```

Comment:
Hm, that's not really a good solution either... Why not calling the `initFieldDescriptors` when it is really needed? Or replace it with a `getFieldDescriptors`-method, which lazily loads?